README.md

```diff
@@ -47,7 +47,9 @@ Clicking on these artifacts will download a ZIP file with the executable. The ex
 Start the executable from the command line, providing the required command line arguments.
 
 ```
-Usage: czicompress [OPTIONS]
+Usage: czicompress.exe [OPTIONS]
+
+  version: 0.4.2.0
```

Comment:
The version does not match 0.5.0 in the CMakeLists.txt

Reply:
Maybe completely remove the version from the Readme?

Reply:
Removing it from the Readme seems like a good idea to me. Done.